clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -279,6 +294,13 @@ public void disconnected(ClientDescriptor clientDescriptor) {
 
   @Override
   public EhcacheEntityResponse invoke(ClientDescriptor clientDescriptor, EhcacheEntityMessage message) {
+    if (this.offHeapResourceIdentifiers.isEmpty()) {
+      return responseFactory.failure(
+          new IllegalStateException("Server started without any offheap resources defined." +
```

Comment:
2 reasons:
1. If I throw any exception from here it'll crash the server.
2. Can't throw `ResourceConfigurationException` which is a checked exception directly from this invoke method.

May be you've mistaken the place from which I'm throwing this. I'm throwing from _invoke()_ not _configure()_.